judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/data/errorType.etl

```diff
@@ -1,30 +1,30 @@
 import "../../../../operations/_importAll.eol";
 
-rule CreateUnmappedErrorTransferObjectType
-    transform s : JSL!ErrorDeclaration
-    to t : JUDOPSM!UnmappedTransferObjectType
-    {
-        t.setId("(jsl/" + s.getId() + ")/CreateUnmappedErrorTransferObjectType");
-        t.name = s.name;
-        t.optional = false;
-        t.queryCustomizer = false;
-
-        s.eContainer.getModelRoot().elements.add(t);
-
-        // By default owned attributes are transformed in `transferAttribute.etl`, therefore we only transform inherited
-        // members by calling the flattened member list from `s.getInheritedFields()`
-        if (s.`extends`.isDefined()) {
-            var inheritedFields = s.getInheritedFields();
-
-            for (inheritedField in inheritedFields) {
-                if (inheritedField.isKindOf(JSL!ErrorField)) {
-                    t.attributes.add(inheritedField.equivalentWithPrefix("CreateTransferAttributeForErrorTransferObjectType", t.getId()));
-                } else {
-                    throw "Unsupported Error Field Type: " + inheritedField.name;
-                }
-            }
-        }
-
-        log.debug("Created ErrorType: " + t.name);
-    }
-
+//rule CreateUnmappedErrorTransferObjectType
+//    transform s : JSL!ErrorDeclaration
+//    to t : JUDOPSM!UnmappedTransferObjectType
+//    {
+//        t.setId("(jsl/" + s.getId() + ")/CreateUnmappedErrorTransferObjectType");
+//        t.name = s.name;
+//        t.optional = false;
+//        t.queryCustomizer = false;
+//
+//        s.eContainer.getModelRoot().elements.add(t);
+//
+//        // By default owned attributes are transformed in `transferAttribute.etl`, therefore we only transform inherited
+//        // members by calling the flattened member list from `s.getInheritedFields()`
+//        if (s.`extends`.isDefined()) {
+//            var inheritedFields = s.getInheritedFields();
+//
+//            for (inheritedField in inheritedFields) {
+//                if (inheritedField.isKindOf(JSL!ErrorField)) {
+//                    t.attributes.add(inheritedField.equivalentWithPrefix("CreateTransferAttributeForErrorTransferObjectType", t.getId()));
+//                } else {
+//                    throw "Unsupported Error Field Type: " + inheritedField.name;
+//                }
+//            }
+//        }
+//
+//        log.debug("Created ErrorType: " + t.name);
+//    }
+//
```

Comment:
Is there any intention to keep? Where is the Error transfer generation?

Reply:
file removed